storage/aptosdb/proptest-regressions/transaction_store/test.txt

```diff
@@ -0,0 +1,7 @@
+# Seeds for failure cases proptest has generated in the past. It is
```

Comment:
we don't usually commit these proptest regression files.

Reply:
Done.